### PR TITLE
Add event download links + composer as dependency manager

### PR DESCRIPTION
**Pull request by mashehu**
Added a dropdown with calendar event links to non-past events, in the list view:
Screenshot 2021-07-07 at 17 24 36
and the details view: 
Screenshot 2021-07-07 at 17 24 18
Open for inputs about the wording and placement of these buttons.

Since I was adding another external library, I decided to switch all php dependencies to be managed by composer (partially addressing #541 with it). Therefore, I added another image to `docker-compose.yml` which runs `compose install` on every start, so one doesn't need to manually install the dependencies locally.
I tested that this doesn't break the implementation of any of the other libraries we use (twitteroauth, parsedown(-extra) and Spyc). But it would be good if somebody else can have another look. 

PS: If we really end up using this we owe the developers a postcard 🙂: https://github.com/spatie/calendar-links#postcardware

### CODE REVIEW

File: includes/parse_md.php [hunk 1]
```diff
@@ -8,8 +8,9 @@
 require_once('functions.php');
 
 // Markdown parsing libraries
-require_once(dirname(__FILE__).'/libraries/parsedown/Parsedown.php');
-require_once(dirname(__FILE__).'/libraries/parsedown-extra/ParsedownExtra.php');
+require "../vendor/autoload.php";
```
Comment (ewels) [inline includes/parse_md.php:11]: Do you need this again as it's already been done in `functions.php`?

**Review by ewels — APPROVED** [review]
Awesome! Composer is another one of those things I've heard about but never actually sat down to figure out. This looks super nice, definitely a good improvement 👍🏻

Calendar additions look fab too. Maybe reword the download link to _Download calendar file_ or something? Don't love that, but to make it a bit more explicitly calendar related. _Download .ics file_?

**Comment by mashehu** [comment]
How about this solution:
Screenshot 2021-07-07 at 21 10 20

File: includes/parse_md.php (hunk 1 above)
Comment (mashehu) [reply includes/parse_md.php]: thanks, I fixed it.

File: public_html/events.php [hunk 2]
```diff
@@ -19,10 +20,35 @@
...
+  $link = Link::create($event['title'], $start, $end)
+    ->description($event['subtitle'] ? $event['subtitle'] : "")
+    ->address($address);
+
+  $event_download_button =  '<div class="dropdown btn-group" role="group">
+          <button   type="button" class="btn '.$button_style.' dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <i class="far fa-calendar-plus mr-1"></i> Export event
+          </button>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="' . $link->ics() . '" target="_blank"> Download as iCalendar</a>
```
Comment (mashehu) [inline public_html/events.php:39]:
```suggestion
            <a class="dropdown-item" href="' . $link->ics() . '" target="_blank"> Download iCal Event</a> 
```
